internal/providers/google/gke_cluster_model.go

```diff
@@ -22,6 +22,13 @@ import (
 	"github.com/jinzhu/gorm"
 )
 
+// TableName constants
+const (
+	GKEClusterModelTableName  = "google_gke_clusters"
```

Comment:
If not used outside of this package, these constants should not be exported.

Reply:
Understood and fixed. Need to get to used to this naming convention :)